build_all.sh

```diff
@@ -20,12 +20,13 @@ conda install -yq boa
 # load gpuci tools
 source ~/.bashrc
 
-conda mambabuild -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
-    recipes/xgboost
+export RAPIDS_CONDA_BLD_ROOT_DIR='/tmp/conda-bld-workspace'
+export RAPIDS_CONDA_BLD_OUTPUT_DIR='/tmp/conda-bld-output'
 
-conda build -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
-    recipes/xgboost --output > $WORKSPACE/conda-output
+conda mambabuild --python=$PYTHON --croot=$RAPIDS_CONDA_BLD_ROOT_DIR --output-folder=$RAPIDS_CONDA_BLD_OUTPUT_DIR recipes/xgboost
```

Comment:
```suggestion
conda mambabuild \
  --python=$PYTHON \
  --croot=$RAPIDS_CONDA_BLD_ROOT_DIR \
  --output-folder=$RAPIDS_CONDA_BLD_OUTPUT_DIR \
  recipes/xgboost
```

let's add some new lines for readability.